third_party/gpus/cuda_configure.bzl

```diff
@@ -0,0 +1,248 @@
+# -*- Python -*-
+
+load(
+    "@bazel_tools//tools/cpp:configure_util.bzl",
+    "find_cc",
+    "get_cxx_inc_directories",
+)
+
+
+_DEFAULT_CUDA_TOOLKIT_PATH = "/usr/local/cuda"
+_DEFAULT_CUDNN_INSTALL_PATH = "/usr/local/cuda"
+
+
+def _cuda_toolkit_path(repository_ctx):
+  """Finds the cuda toolkit directory."""
+  cuda_toolkit_path = _DEFAULT_CUDA_TOOLKIT_PATH
+  if "CUDA_TOOLKIT_PATH" in repository_ctx.os.environ:
```

Comment:
Sorry yes it is better not to use the async version.

Grrr. It is really hard to follow a code-review on that stuff.